be/src/olap/tablet.cpp

```diff
@@ -1531,4 +1532,24 @@ std::shared_ptr<MemTracker>& Tablet::get_compaction_mem_tracker(CompactionType c
     }
 }
 
+Status Tablet::get_dict_data(std::set<std::string>& dict_words, int col_id) {
+    std::vector<BetaRowset*> beta_row_sets;
```

Comment:
Do not use raw ptr here. Because if you use raw ptr, the rowset maybe released before you use it. May core!!!!